sdk/core/azure-core/src/main/java/com/azure/core/implementation/util/ReferenceManagerImpl.java

```diff
@@ -25,42 +28,84 @@ public final class ReferenceManagerImpl implements ReferenceManager {
     // Base name for ResourceManager threads.
     private static final String BASE_THREAD_NAME = "azure-sdk-referencemanager";
 
+    private static final Object CLEANER;
+    private static final MethodHandle CLEANER_REGISTER;
+
+    static {
```

Comment:
Remove `there is another Java 9 specific implementation in
/src/main/java9 for multi-release JARs.` from class javadoc.